app/forms/partner_details_form.rb

```diff
@@ -1,9 +1,9 @@
-class PartnerForm
+class PartnerDetailsForm
   include ActiveModel::Model
   include ActiveModel::Attributes
   include SessionPersistable
 
-  ATTRIBUTES = %i[partner].freeze
+  ATTRIBUTES = %i[partner_over_60 partner_employed dependants].freeze
```

Comment:
Careful - because this model includes `SessionPersistable`, not `SessionPersistableForPartner`, the dependants attribute here will be saved to the session as `dependants`, overwriting the _client_ dependants attribute.

I would recommend including the partner persistable concern, so this line then becomes `ATTRIBUTES = %i[over_60 employed dependants]` and we know that `partner_` will get prepended to the relevant session. keys.